src/ServiceControl.Audit/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -16,6 +16,8 @@ public static Task<TimerJobExecutionResult> RunCleanup(int deletionBatchSize, Do
 
             logger.Debug("Trying to find expired ProcessedMessage and SagaHistory documents to delete (with threshold {0})", threshold.ToString(Default.DateTimeFormatsToWrite, CultureInfo.InvariantCulture));
             AuditMessageCleaner.Clean(deletionBatchSize, database, threshold, token);
+            logger.Debug("Trying to find expired KnownEndpoints to delete (with threshold {0})", threshold.ToString(Default.DateTimeFormatsToWrite, CultureInfo.InvariantCulture));
```

Comment:
Why have a separate log line for this and not just include it in the one above? The threshold is the same

Reply:
I stuck to the pattern started in

https://github.com/Particular/ServiceControl/blob/5163de59031cfea1fbd31e4352b8af0375c9ef53/src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs#L17
https://github.com/Particular/ServiceControl/blob/5163de59031cfea1fbd31e4352b8af0375c9ef53/src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs#L22
https://github.com/Particular/ServiceControl/blob/5163de59031cfea1fbd31e4352b8af0375c9ef53/src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs#L29

but even there it is kind of not the same because ProcessedMessages and SagaHistory are in one log message. Happy to change